usage/index.adoc

```diff
@@ -74,4 +91,26 @@ If you want to inspect the source code for all of the above it can be
 found
 https://github.com/jbosstools/jbosstools-base/tree/master/usage[here]
 
+== Error Reporting
+
+The error reporting is using https://wiki.eclipse.org/EPP/Logging[Eclipse AERI] which
+been successfully used by Eclipse Foundation to gather error reports from core Eclipse
+plugins.
+
+The error reporting is available when installing most JBoss Tools plugins after JBoss Tools 4.3.2.
```

Comment:
If you select the JBoss Tools Usage Reporting feature when installing JBoss Tools, AERI reporting will be enabled. This functionality was added as of JBoss Tools 4.3.1.